`saxon-js` dependency not declared in `package.json` causing `Cannot find module` error

## What happens

`src/commands/docs.js:8` requires `saxon-js` directly:

```js
const SaxonJS = require('saxon-js');
```

`saxon-js` is not listed in `package.json`'s `dependencies`. It currently resolves only because it happens to be pulled in transitively through `eo2js` (`eo2js@0.0.8 → saxon-js@2.7.0`), confirmed with `npm ls saxon-js`.

By contrast, `marked`, which is also used only by `docs.js`, is correctly declared as a direct dependency — so the two modules `docs.js` depends on are handled inconsistently.

Verified: removing `node_modules/saxon-js` and running `eoc docs` produces a raw, uncaught error:
```
Error: Cannot find module 'saxon-js'
```
with a full stack trace and exit code 1 — not a controlled CLI error message.

Because the dependency is undeclared, it can silently disappear in three realistic scenarios: a future `eo2js` version drops or renames its own `saxon-js` dependency, a package manager that enforces strict `node_modules` isolation (e.g. `pnpm`, or npm with `--legacy-peer-deps` off in some configurations) refuses to hoist a transitive dependency this way, or `eo2js` itself is ever swapped out.

## What should happen

`saxon-js` should be added to `package.json`'s own `dependencies`, matching how `marked` is already declared, so `eoc docs` does not depend on an implementation detail of a different dependency's own dependency tree.

## Steps to reproduce

1. In a checkout of `eoc`, run `npm install` then `rm -rf node_modules/saxon-js`.
2. Run `node src/eoc.js docs`.
3. Observe an uncaught `Error: Cannot find module 'saxon-js'` instead of `docs` working (since `eo2js`, the package that actually depends on it, is still installed and unaffected).

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "fast-xml-parser": "^5.10.1",
     "marked": "^4.3.0",
     "relative": "^3.0.2",
+    "saxon-js": "^2.7.0",
     "semver": "^7.7.2",
     "sync-request": "^6.1.0",
     "xmlhttprequest": "^1.8.0"

--- a/test/commands/test_docs.js
+++ b/test/commands/test_docs.js
@@ -370,4 +370,13 @@ describe('docs', () => {
     );
     done();
   });
+  /**
+   * Tests that 'saxon-js', required directly by this module, is declared
+   * as its own dependency instead of relying on it being pulled in
+   * transitively through 'eo2js'.
+   */
+  it('declares saxon-js as a direct dependency', () => {
+    const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, '..', '..', 'package.json'), 'utf-8'));
+    assert.ok(pkg.dependencies['saxon-js'], 'saxon-js must be listed in package.json dependencies');
+  });
 });

--- a/test/commands/test_docs.js
+++ b/test/commands/test_docs.js
@@ -371,12 +371,22 @@ describe('docs', () => {
     done();
   });
   /**
-   * Tests that 'saxon-js', required directly by this module, is declared
-   * as its own dependency instead of relying on it being pulled in
-   * transitively through 'eo2js'.
+   * Tests that every package 'docs.js' requires is declared by 'eoc' itself
+   * and resolves, instead of being reached transitively through 'eo2js'.
    */
-  it('declares saxon-js as a direct dependency', () => {
+  it('declares every package it requires', () => {
+    const source = path.join(__dirname, '..', '..', 'src', 'commands', 'docs.js');
     const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, '..', '..', 'package.json'), 'utf-8'));
-    assert.ok(pkg.dependencies['saxon-js'], 'saxon-js must be listed in package.json dependencies');
+    const required = [...fs.readFileSync(source, 'utf-8').matchAll(/require\('(?<name>[^.'][^']*)'\)/g)]
+      .map((match) => match.groups.name)
+      .filter((name) => !require('module').builtinModules.includes(name));
+    assert.notStrictEqual(required.length, 0, 'no external requires were found in docs.js');
+    required.forEach((name) => {
+      assert.ok(pkg.dependencies[name], `${name} is required by docs.js but not in package.json dependencies`);
+      assert.doesNotThrow(
+        () => require.resolve(name, {paths: [path.dirname(source)]}),
+        `${name} is declared but does not resolve`
+      );
+    });
   });
 });

